extensions/positron-python/src/client/positron/util.ts

```diff
@@ -26,3 +28,15 @@ export async function whenTimeout<T>(ms: number, fn: () => T): Promise<T> {
     await delay(ms);
     return fn();
 }
+
+/**
+ * Retrieves the user's default Python interpreter path from VS Code settings
+ *
+ * @returns The configured Python interpreter path if it exists and is not 'python',
+ *          otherwise returns an empty string
+ */
+export function getUserDefaultInterpreter(): string {
+    const configuration = vscode.workspace.getConfiguration('python');
+    const defaultInterpreterPath = configuration?.get<string>('defaultInterpreterPath');
+    return !defaultInterpreterPath || defaultInterpreterPath === 'python' ? '' : defaultInterpreterPath;
```

Comment:
`'python'` is the default for this setting, so we'll only use the setting if it has been changed

Reply:
Might be worth adding that as a comment?